mlflow/langchain/runnables.py

```diff
@@ -117,7 +117,8 @@ def _load_runnable_with_steps(file_path: Union[Path, str], model_type: str):
         )
 
     steps = {}
-    for step in os.listdir(steps_path):
+    # ignore hidden files
+    for step in filter(lambda x: not x.startswith("."), os.listdir(steps_path)):
```

Comment:
```suggestion
    for step in (f for f in os.listdir(steps_path) if not f.startswith(".")):
```

Ref: https://docs.astral.sh/ruff/rules/unnecessary-map/

Reply:
not sure if comprehension is really efficient, but I usually use comprehension when it can accomplish what I want.